starlite/plugins/sql_alchemy/plugin.py

```diff
@@ -329,13 +334,18 @@ def parse_model(model_class: DeclarativeMeta) -> Mapper:
             model_class: An SQLAlchemy declarative class.
 
         Returns:
-            An SQLAlchemy inspection 'Mapper'.
+            A SQLAlchemy `Mapper`.
         """
-        if not isinstance(model_class, DeclarativeMeta):
-            raise ImproperlyConfiguredException(
-                "Unsupported 'model_class' kwarg: only subclasses of the SQLAlchemy `DeclarativeMeta` are supported"
-            )
-        return inspect(model_class)
+        try:
+            inspected = inspect(model_class)
+        except NoInspectionAvailable:
+            pass
```

Comment:
Shouldbt we raise here?

Reply:
It does raise, `pass` here skips the else block and raises the exception below.